examples/ffi/exporter.cpp

```diff
@@ -114,7 +114,7 @@ int main(int argc, char *argv[]) {
 
```

Comment:
:arrow_up: Would it perhaps make sense to update the `Profile_add` call above to check the result as well?

Reply:
Yes, it does. The compiler didn't whine at me, so I added `#[must_use]`. Now it yells at me, and I fixed it in the C example too.